src/fastoad/notebooks/02_explanation/05_FAST-OAD.ipynb

```diff
@@ -0,0 +1,441 @@
+{
```

Comment:
<blockquote>variable_viewer</blockquote><p><code>variable_viewer</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Changed in dfe233f7

Reply:
<p>Really?</p>

Reply:
Changed in 77d98d3c ? I only did one of the 2 in the previous commit, mb